cmd/zapi/cmd/intake/ls.go

```diff
@@ -0,0 +1,51 @@
+package intake
+
+import (
+	"flag"
+
+	"github.com/brimsec/zq/cli/outputflags"
+	"github.com/brimsec/zq/cmd/zapi/cmd"
+	"github.com/mccanne/charm"
+)
+
+var Ls = &charm.Spec{
+	Name:  "ls",
+	Usage: "intake ls [-l]",
+	Short: "list intakes",
+	Long:  ``,
```

Comment:
If no long comment considering dropping this?

Reply:
I updated the descriptions, adding some & removing the blank ones, in bcf96dd9579f879c5bc205bc621c7021cb8b0128 .